### PR TITLE
0.2.0 documentation

### PR DESCRIPTION
**Pull request by leogdion**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,417 @@
...
+  func start () {
+    self.connectivityObserver.start(queue: .global())
+  }
+}
+```
+
+There are 3 important pieces:
+
+1. The `NetworkObserver` called `connectivityObserver`
+2. On `init`, we use `Combine` to listen to the publisher and store the each new `pathStatus` to our `@Published` property.
```
Comment (mgnt) [inline README.md:114]: store ~the~ each

File: README.md [hunk 2]
```diff
@@ -1 +1,417 @@
...
+3. A `start` method which needs to be called to start listening to the `NetworkObserver`.
+
+Therefore for our `SwiftUI` `View`, we need to `start` listening `onAppear` and can use the `pathStatus` property in the `View`:
```
Comment (mgnt) [inline README.md:117]: Does any cleanup need to happen in onDisappear?

File: README.md [hunk 3]
```diff
@@ -1 +1,417 @@
...
+    IpifyPing(session: .shared, timeInterval: 10.0)
+   )
+```
+
+## Communication between iPhone and Apple Watch
+
+Besides networking, **SundialKit** also provides an easier reactive interface into `WatchConnectivity`. This includes:
+
+1. Various connection statues like `isReachable`, `isInstalled`, etc..
+2. Send messages between the iPhone and paried Apple Watch
```
Comment (mgnt) [inline README.md:182]: ~paried~ paired

File: README.md [hunk 4]
```diff
@@ -1 +1,417 @@
...
+    // use the shared `URLSession` and check every 10.0 seconds
+    IpifyPing(session: .shared, timeInterval: 10.0)
+   )
+```
+
+## Communication between iPhone and Apple Watch
+
+Besides networking, **SundialKit** also provides an easier reactive interface into `WatchConnectivity`. This includes:
+
+1. Various connection statues like `isReachable`, `isInstalled`, etc..
```
Comment (mgnt) [inline README.md:181]: ~statues~ statuses

File: README.md (hunk 2 above)
Comment (leogdion) [reply README.md:117]: you could `cancel` I suppose if it's no longer needed but in many cases it may just be run as a singleton. I am to opinions.

**Pull request merged**